packages/core/src/types/IActor.ts

```diff
@@ -20,4 +21,6 @@ export type ActorEntry = {
   referenceId?: string;
   /** slab thickness for the actor */
   slabThickness?: number;
+  /** the vtkImage data related to an ImageActor */
+  imageData?: vtkImageData;
```

Comment:
I don't think this belongs here. We have the actor, you can grab the imageData directly from it

`actor.getMapper().getInputData()``